spec/integration/meal_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe 'Meal actions', type: :feature do
+  before :each do
+    @user = User.create(name: 'Test User', email: 'clint@gmail.com', password: 'password')
+    @meal = Meal.create(name: 'Test Meal', author_id: @user.id, icon: 'https://www.flaticon.com/svg/static/icons/svg/1046/1046784.svg')
+    sign_in @user
+  end
+
+  describe 'Meal index page' do
+    it "displays the meal's name" do
+      visit meals_path
+      expect(page).to have_content(@meal.name)
+    end
+
+    it "displays the meal's icon" do
+      visit meals_path
+      expect(page).to have_css("img[src*='1046784.svg']")
+    end
+  end
```

Comment:
Great job @ClintonYounge 🤝

- [ ] Kindly fix the failing test so that your tests can all pass the checks.
![failed test](https://github.com/ClintonYounge/Meals-Budgeting-App/assets/63917725/5f15448f-b164-410a-9b7a-a672d70d2abc)